command_test.go

```diff
@@ -4311,6 +4311,9 @@ func TestCommandReadArgsFromStdIn(t *testing.T) {
 				},
 				&StringSliceFlag{
 					Name: "ssf",
+					Config: StringConfig{
+						TrimSpace: true,
+					},
```

Comment:
Is it possible to make `TrimSpace` a first level option of `StringSliceFlag`?

Reply:
@dearchap @abitrolly  Following up on this 🙏